### DISCUSSION TITLE
Any update about Android version

### DISCUSSION THREAD

**Discussion by defencedog** [mifi/lossless-cut#1331] [General]
Previously the good author detailed limitation of Android system. We have seen this application running on Android via `proot + termux` ...But  author also said that project files can be generated on **native** Andorid & the user can process when PC is available.

I am asking about the latter option?

**Comment by mifi** [comment]
For native android/ios version, please see #1331

An alternative is to install ubuntu on android and run LosslessCut arm version, see #1056

**Reply by darinhouston** [reply]
I don't see anything on this

**Reply by mifi** [reply]
Oops i linked the wrong issue. Correct one:
#818
